src/WebJobs.Extensions.DurableTask/Listener/DurableTaskListener.cs

```diff
@@ -2,25 +2,42 @@
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
 using System;
+using System.Collections.Generic;
+using System.Linq;
 using System.Threading;
 using System.Threading.Tasks;
+using DurableTask.AzureStorage.Monitoring;
 using Microsoft.Azure.WebJobs.Host.Executors;
 using Microsoft.Azure.WebJobs.Host.Listeners;
+#if NETSTANDARD2_0
+using Microsoft.Azure.WebJobs.Host.Scale;
+#endif
 
 namespace Microsoft.Azure.WebJobs.Extensions.DurableTask
 {
+#if !FUNCTIONS_V1
```

Comment:
I wonder if using a partial class would make this cleaner? Is it that the same code is being compiled for both v1 and v2, rather than having separate branches for v1/v2?